### PR TITLE
Feature-windows_support

### PR DESCRIPTION
**Pull request by CaptainDario**
This PR should add windows support.

Environment:
* Windows 10

Changes:
* Load TF Lite binary
* update examples (same as macos, linux) to support windows

Notes:
* Camera is disabled for the examples on desktop as this is not really supported

Progress:
- [x] load TF Lite on windows
- [x] test examples
  - [x] audio classification
    - uses some platform channels that do not work on windows
  - [x] image classification mobilenet
    - only from file works
  - [x] live object detection ssd mobile net
    - uses camera package that only works on mobile 
  - [x] object detection ssd mobile net
    - only from file works
  - [x] style transfer
    - works 
  - [x] super resolution esrgan
    - works
  - [x] text classification
    - works

### CODE REVIEW

**Comment by moshOntong-IT** [comment]
Why this become draft sir?

**Comment by CaptainDario** [comment]
Because it is not done.

**Comment by CaptainDario** [comment]
@PaulTR This is now working for me locally.

One thing that should be added is how to compile the TF Lite binary on Windows. Following the procedure linked in the README I only get a debug binary. @PaulTR @moshOntong-IT any ideas?

**Comment by CaptainDario** [comment]
@PaulTR many of the new examples added after the macos pr have quite a few problems when running `flutter analyze .`

**Comment by PaulTR** [comment]
@st-duymai Hey Duy, would you mind checking into the `flutter analyze` issue on the samples? This PR is also ready for testing it looks like.

Thanks!

**Comment by st-duymai** [comment]
Hi @PaulTR , @CaptainDario Since this project uses Melos to manage project, you must run `dart pub global activate melos` and `melos bootstrap` before `flutter analyze .`. Run `flutter analyze .` without melos can causes missing dependencies and locally linking package issue.

> This PR is also ready for testing it looks like.

I will test it tomorrow on my Windows machine at home.

**Comment by st-duymai** [comment]
@PaulTR @CaptainDario I have tested all of the above apps, and they all work except for the features related to the camera and recording audio. Two projects, live_object_detection_ssd_mobile_net and audio_classification, do not work completely because they are not implemented for Windows. I think that we should remove the Windows package from these two projects.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,27 @@ For this, first a `.dylib` needs to be built. You can follow the [Bazel build gu
...
+### Windows
+
+For Windows a TensorFlow Lite dynamic library needs to be added to the project manually.
+For this, first a `.dll` needs to be built. You can follow the [Bazel build guide](https://www.tensorflow.org/lite/guide/build_arm) or the [CMake build guide](https://www.tensorflow.org/lite/guide/build_cmake) to build the libraries.
+
+As a second step, the library needs to be added to your application's project. This is a simple procedure
+
+1. Create a folder called `blobs` in the top level of your project
+2. Copy the `libtensorflowlite_c-win.dll` to this folder
+3. Append following lines to your `linux/CMakeLists.txt`
```
Comment (st-duymai) [inline README.md:101]: I think it should be windows
```suggestion
3. Append following lines to your `windows/CMakeLists.txt`
```

**Comment by PaulTR** [comment]
> @PaulTR @CaptainDario I have tested all of the above apps, and they all work except for the features related to the camera and recording audio. Two projects, live_object_detection_ssd_mobile_net and audio_classification, do not work completely because they are not implemented for Windows. I think that we should remove the Windows package from these two projects.

Sounds good. Yeah camera isn't expected to work for desktop (just isn't supported generally by Flutter from what I understand). I'm not sure if a similar limitation exists for audio - if audio can be supported, that'd be awesome, but I'm not too worried about it, so cleaning that up would be perfect and then we can get this merged. 

Thanks for the heads on on Melos!

As for the binaries, I'm not sure if there's a difference in release vs debug ones? I'm only aware of the one set of docs for creating them. If there's something blocking with that though I can definitely bubble it up to eng.

File: README.md (hunk 1 above)
Comment (CaptainDario) [reply README.md]: yes, you are right, I changed it. Thanks!

**Comment by CaptainDario** [comment]
With the audio example, I do not know how it works. It is calling some platform code that does not work on Mac, Linux, Windows. However, I added to each README which platforms are supported, I thought that should work.

Example for audio:

image


In regards to the binaries, for me the confusing thing is, that on Mac and Linux it does *not* create a debug binary by default. I think it is really confusing, but maybe that's only me.

**Pull request merged**